terraform/aws/kube/kube.tf

```diff
@@ -0,0 +1,202 @@
+provider "kubernetes" {
+  load_config_file = "false"
+
+  host = var.kube_cluster_endpoint
+
+  cluster_ca_certificate = base64decode(var.kube_cert_data)
+
+  config_context_auth_info = "aws"
+  config_context_cluster   = "kubernetes"
+
+  exec {
+    api_version = "client.authentication.k8s.io/v1alpha1"
+    command     = "aws-iam-authenticator"
+    args        = ["token", "-i", "wandb"]
+  }
+}
+
+##########################################
+# Variables
+##########################################
+
+variable "license" {
+  description = "The license string for your local instance."
+  type        = string
```

Comment:
In my tests on <=0.9.29 this didn't work, if it works now AWESOME.  If not, we could pair for < 15 minutes to get to the bottom of it 😎

Reply:
It worked for me but I only tried this version of the stack once. Where did it go wrong for you?

Reply:
If the env tarball doesn't exist in bucket I didn't think it worked on initial boot, but I'm not positive.

Reply:
i did the full apply again and it worked, but im using 0.9.30